cumulus/polkadot-parachain/src/main.rs

```diff
@@ -26,6 +26,9 @@ mod cli;
 mod command;
 mod rpc;
 
-fn main() -> sc_cli::Result<()> {
-	command::run()
+fn main() {
+	if let Err(error) = command::run() {
```

Comment:
Why? To use the `Display` implementation? Can we not make `Debug` use `Display` internally?